data/surface.tablet

```diff
@@ -0,0 +1,14 @@
+# Microsoft Surface Pro/Surface Book
+
+[Device]
+Name=Microsoft Surface Pro/Book
+DeviceMatch=mei:1b96:006a;mei:1b96:001F;mei:1b96:005e;mei:1b96:0020
```

Comment:
lowercase hex only please (I think the CI should pick this up now that I've triggered it)